.gitlab-ci-check-helm-version-bump.yml

```diff
@@ -0,0 +1,109 @@
+# .gitlab-ci-check-helm-version-bump.yml
+#
+# This gitlab-ci template updates the Helm Chart versions
+# for a given container
+#
+# Requires DOCKER_REPOSITORY variable to be set in the calling Pipeline.
+# Add it to the project in hand through Gitlab's include functionality
+# variables:
+#   DOCKER_REPOSITORY: <Image FQN, i.e mendersoftware/reponame>
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-helm-version-bump.yml
+# 
+# It requires that the upstream pipeline specifies:
+#
+# - CONTAINER: alvaldi-gui # the container name that later will be translated into
+#                            values.yaml reference
+# - DOCKER_PUBLISH_COMMIT_TAG: ${CI_COMMIT_REF_NAME}_${CI_COMMIT_SHA} 
+# - SYNC_ENVIRONMENT: [staging|prod] # the values-${SYNC_ENVIRONMENT}.yaml to be updated
+# - CHART_DIR: mender # the Helm Chart directory in the helm repo
+# - HELM_PATCH_VERSION: ${CI_PIPELINE_ID} # the Chart.yaml version - use Semver format
+#                                         # prerelases have to be tagged with ${CI_PIPELINE_ID}-staging
+#                                         # to avoid releasing a prerelease to prod (stable semver releases only)
+#
+# The non-production version updates the Chart.yaml with a prerelease version:
+# version: x.y.${CI_PIPELINE_ID}-staging
+# On the downstream, FluxCD is configured to trigger a new Helm deployment every releases and pre-releases on the non-prod envs.
+# At the same way, the production envs are configured to trigger only stable releases.
+
+stages:
+  - version-bump
+
+helm-version-bump:
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "pipeline"
+  stage: version-bump
+  image: registry.gitlab.com/northern.tech/mender/mender-test-containers:aws-k8s-v1-master
```

Comment:
I am unable to check if that image exists:
```
# docker pull registry.gitlab.com/northern.tech/mender/mender-test-containers:aws-k8s-v1-master
Error response from daemon: Head "https://registry.gitlab.com/v2/northern.tech/mender/mender-test-containers/manifests/aws-k8s-v1-master": denied: access forbidden
```

Reply:
You should login to the gitlab container registry with your own token: https://docs.gitlab.com/ee/user/packages/container_registry/authenticate_with_container_registry.html

Reply:
I am logged in.

Reply:
but if it pulls for you, then perhaps it is something else and you do not have to worry about it.